cepgen.spec

```diff
@@ -0,0 +1,39 @@
+### RPM external cepgen 1.2.1patch1_gcc700
+
+Source: https://github.com/cepgen/cepgen/archive/refs/tags/%{realversion}.tar.gz
+Patch: cepgen_nopython_noroot
+
+BuildRequires: cmake gmake
+Requires: gsl OpenBLAS hepmc lhapdf pythia6 bz2lib zlib xz
+
+%prep
+%setup -n %{n}-%{realversion}
+%patch -p1
+
+%build
+rm -rf ../build
+mkdir ../build
+cd ../build
+
+export GSL_DIR=${GSL_ROOT}
+export OPENBLAS_DIR=${OPENBLAS_ROOT}
+export HEPMC_DIR=${HEPMC_ROOT}
+export LHAPDF_PATH=${LHAPDF_ROOT}
+export PYTHIA6_DIR=${PYTHIA6_ROOT}
+
+cmake ../%{n}-%{realversion} \
+  -DCMAKE_INSTALL_PREFIX:PATH="%i" \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_PREFIX_PATH="${BZ2LIB_ROOT};${ZLIB_ROOT};${XZ_ROOT}"
+
+make %{makeprocesses}
+
+%install
+cd ../build
+make install
```

Comment:
@smuzaffar I realise I unintentionally dropped the `%{makeprocesses}` when replacing Ninja to gmake. Could this be the reason the headers are not installed properly?

Reply:
No that is just to install in parallel. 10.6.X has old cmake and you might have something in your cmake configuration with does not work properly with old cmake. cmake has properly installed `bin`, `lib`, and `share` part but for include it somehow has full path e.g. install_manifest.txt file in build directory has
```
/build/muz/cep/w/tmp/BUILDROOT/b31440bef06979bdfb817bec57e15cb8/opt/cmssw/slc7_amd64_gcc700/external/cepgen/1.2.1patch1_gcc700-b31440bef06979bdfb817bec57e15cb8/lib64/libCepGen.so
/build/muz/cep/w/tmp/BUILDROOT/b31440bef06979bdfb817bec57e15cb8/opt/cmssw/slc7_amd64_gcc700/external/cepgen/1.2.1patch1_gcc700-b31440bef06979bdfb817bec57e15cb8/include//build/muz/cep/w/BUILD/slc7_amd64_gcc700/external/cepgen/1.2.1patch1_gcc700-b31440bef0
6979bdfb817bec57e15cb8/cepgen-1.2.1patch1_gcc700/Physics/Beam.h

```

i.e. lib has correct path but header has `install-prefix/build-dir/header`